src/components/atoms/ActionButtonV2/ActionButtonV2.js

```diff
@@ -0,0 +1,63 @@
+import styles from '!!raw-loader!./ActionButtonV2.css';
+import varStyles from '!!raw-loader!../../../shared/variables.css';
+import bootstrapStyles from '!!raw-loader!../../../shared/themed-bootstrap.css';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<div class="action-button-container">
+  <a class="btn" role="button" href="">
+    <div class="abutton">
+      <div class="top-icon">
+        <cod-icon data-icon="" data-size="x-large">
```

Comment:
It's a little different. Unlike `Icon.js`, there's no reason to expose `is-highlighted` as an attribute on `cod-action-button-v2`. Instead, we can just statically set the attribute on the `cod-icon` in the template here:

`<cod-icon data-icon="" data-size="x-large" is-highlighted>`